static/main.ts

```diff
@@ -644,6 +644,22 @@ function start() {
         sizeCheckNavHideables();
     }
 
+    new clipboard('.btn.clippy');
+
+    const [themer, settings] = setupSettings(hub);
```

Comment:
This looks like it's not an issue to push it up-hear, good idea :+1:

Reply:
Just had to use `settings` in the new code